apple/internal/rule_factory.bzl

```diff
@@ -137,6 +137,11 @@ _COMMON_PRIVATE_TOOL_ATTRS = dicts.add(
             allow_single_file = True,
             default = Label("@build_bazel_rules_apple//apple/internal/templates:ios_sim_template"),
         ),
+        "_macos_runner_template": attr.label(
+            cfg = "host",
+            allow_single_file = True,
+            default = Label("@build_bazel_rules_apple//apple/internal/templates:macos_sim_template"),
```

Comment:
Sim in the file name probably doesn't make sense here

Reply:
Thanks! Will change

Reply:
Removed, thanks!